localstack/services/apigateway/provider.py

```diff
@@ -1836,6 +1836,8 @@ def get_export(
             export_type=export_type,
             export_format=accepts,
             with_extension=has_extension,
+            region_name=context.region,
+            account_id=context.account_id,
```

Comment:
Nitpick: to keep the codebase consistent, prefer having the `account_id` arg before the `region_name`.